homeassistant/components/nut/sensor.py

```diff
@@ -57,12 +57,15 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
         if sensor_type in status or (
             sensor_type == KEY_STATUS_DISPLAY and KEY_STATUS in status
         ):
+            sensor_description = next(
```

Comment:
I don't think this is an improvement. Without the dict container for the entity descriptions we need to iterate the container just to pick one description.

Reply:
@frenck I did this change based on your https://github.com/home-assistant/core/pull/54269#discussion_r686858749 
I agree with @MartinHjelmare , since honestly I do not see any other possibility to always iterate over all available descriptions, to pick the actual needed one 🤔

Reply:
ok, I've removed the commit containing the change from dict to tuple